data/campaigns/Dead_Water/scenarios/04_Slavers.cfg

```diff
@@ -721,7 +722,7 @@
 
         [message]
             speaker=Kai Krellis
-            message= _ "So you will help us defeat them?"
+            message= _ "So, will you help us defeat them?"
```

Comment:
I think I prefer the old phrasing here. It doesn't have any grammatical issues, so I'm not sure why you changed it? It's also a stronger question, giving an implication that he expects a "yes" answer, which kind of befits a king; whereas your change makes it a more neutral question.